README.md

```diff
@@ -89,15 +89,16 @@ $ minikube addons disable metrics-server
 
 ### Kubernetes compatibility matrix
 
-| kube-prometheus stack | Kubernetes 1.14 | Kubernetes 1.15 | Kubernetes 1.16 | Kubernetes 1.17 |
-|-----------------------|-----------------|-----------------|-----------------|-----------------|
-| `release-0.3`         | ✔              | ✔              | ✔              | ✔              |
-| `release-0.4`         | ✗              | ✗              | ✔              | ✔              |
-| `HEAD`                | ✗              | ✗              | ✗              | ✔              |
+| kube-prometheus stack | Kubernetes 1.14 | Kubernetes 1.15 | Kubernetes 1.16 | Kubernetes 1.17 | Kubernetes 1.18 |
+|-----------------------|-----------------|-----------------|-----------------|-----------------|-----------------|
+| `release-0.3`         | ✔              | ✔              | ✔              | ✔              | ✗
+| `release-0.4`         | ✗              | ✗              | ✔              | ✔              | ✗
+| `release-0.5`         | ✗              | ✗              | ✗              | ✔              | ✔
+| `HEAD`                | ✗              | ✗              | ✗              | ✗              | ✔
```

Comment:
Do we really want to drop support for `1.17` in master?

Reply:
I think its not dropping support just more dropping testing this version. I can rephrase the compatibility matrix to include a maybe it works 🤷‍♀️

Reply:
"We know these are supported and work as we test against these versions, but other versions might work." WDYT?

Reply:
Sounds good to me

Reply:
sgtm